cloudwatch/cloudwatch.go

```diff
@@ -440,19 +446,15 @@ func (output *OutputPlugin) setGroupStreamNames(e *Event) {
 	s := &sanitizer{sanitize: sanitizeGroup, buf: output.bufferPool.Get()}
 
 	if _, err := parseDataMapTags(e, logTagSplit, output.logGroupName, s); err != nil {
-		logrus.Errorf("[cloudwatch %d] parsing log_group_name template: %v", output.PluginInstanceID, err)
-	}
-
-	if e.group = s.buf.String(); len(e.group) == 0 {
-		e.group = output.logGroupName.String
-	}
-
-	if len(e.group) > maxGroupStreamLength {
+		e.group = output.defaultLogGroupName
+		logrus.Errorf("[cloudwatch %d] parsing log_group_name template '%s': %v",
+			output.PluginInstanceID, output.logGroupName.String, err)
```

Comment:
What is setting `err` here? 

Reading the code above this error log, I expected it to be something like "generated log group name is longer than max allowed value, using default group"

Reply:
Sorry I misread the code